### DISCUSSION TITLE
Improved display of recent items in project menu.

### DISCUSSION THREAD

**Discussion by ariebh** [posit-dev/positron#5287] [Feedback]
It can be difficult to identify projects in the recently used section of the project menu when it is stored at a location with a long path. The width of the menu restricts how much of the path can be displayed, cutting of the project's enclosing folder.

I'll often see something like this **C:\Users\Username\Documents\a\very\long\path\to\the\...** when attemting to access a project at **C:\Users\Username\Documents\a\very\long\path\to\the\project\project-name**. Truncating the path from the start rather than the end would improve this. **...\path\to\the\project\project-name**

Preferred behaviour would be closer to that of the project menu in RStudio. In RStudio, recent items are displayed using project names, with paths shown only if there is a naming conflict in menu. 

I'm unsure if this has already been discussed, or if I should create an issue for this.

**Comment by juliasilge** [comment]
Thanks for this report! Can you share what you mean by "project menu" in this case? A screenshot would be super helpful.

**Comment by ariebh** [comment]
Sorry, I wasn't entirely sure what it is called. 

![image](https://github.com/user-attachments/assets/12706c04-e4d3-4e32-9149-833022e301ae)

I mean the menu that is opend when you click on the button with the folder icon in this screen shot.

**Reply by juliasilge** [reply]
Ah thank you! I have opened https://github.com/posit-dev/positron/issues/5301 to track and prioritize this work.
